lib/marathon.js

```diff
@@ -99,6 +105,24 @@ function Marathon(url, opts) {
     };
 
     client.apps = client.app;
+    client.events.createEventSource = function createEventSource(opts) {
```

Comment:
It's better to move this implementation into `lib/events.js`, because this file is kind of "assembly point", and implementation of any method here looks like a violation of responsibility separation (by abstraction levels).